third_party/tsl/third_party/gpus/cuda/hermetic/BUILD

```diff
@@ -0,0 +1,32 @@
+cc_library(
+    name = "headers",
+    hdrs = [
+        "include/builtin_types.h",
+        "include/channel_descriptor.h",
+        "include/cooperative_groups.h",
+        "include/cooperative_groups/**",
+        "include/cuComplex.h",
+        "include/cuda.h",
+        "include/cuda_bf16.h",
+        "include/cuda_device_runtime_api.h",
+        "include/cuda_fp16.h",
+        "include/cuda_fp16.hpp",
+        "include/cuda_fp8.h",
+        "include/cuda_fp8.hpp",
+        "include/cuda_occupancy.h",
+        "include/cuda_runtime.h",
+        "include/cuda_runtime_api.h",
+        "include/device_types.h",
+        "include/driver_types.h",
+        "include/host_defines.h",
+        "include/library_types.h",
+        "include/surface_types.h",
+        "include/texture_types.h",
+        "include/vector_functions.h",
+        "include/vector_types.h",
+    ],
```

Comment:
This is a subset of the headers provided by cuda_cudart, but I'm guessing that's intentional here?  I do see this list repeated (plus a bunch of the ones otherwise omitted from this list) in the cuda_cudart headers section.  Can there should be a single source of truth?

Reply:
Thanks for catching this, I've removed this target - it's not needed here.